newIDE/app/src/InAppTutorial/InAppTutorialTooltipDisplayer.js

```diff
@@ -154,7 +154,11 @@ function InAppTutorialTooltipDisplayer({
         {({ TransitionProps }) => (
           <>
             <Fade {...TransitionProps} timeout={{ enter: 350, exit: 0 }}>
-              <Paper style={{ ...styles.paper, backgroundColor }} elevation={4}>
+              <Paper
+                style={{ ...styles.paper, backgroundColor }}
+                elevation={4}
+                background="light"
```

Comment:
This should be removed since the tooltips are a bit out of theme. Unless the props is mandatory

Reply:
after thought, it's probably a better idea to use the default MuiPaper in this component